### ISSUE TITLE
Invalid @online message format

### ISSUE THREAD

**Issue by cm0x4D** [cloudio-project/cloudio-endpoint-python#10]
In `@online` messages some objects contain a field `_internal` that is not conform to the JSON format and causes the server to drop the message without processing. Only `objects `, `conforms ` and `attributes ` fields are allowed in an such an object. 

As a result the endpoints are not listed in the database at all.

**Comment by cm0x4D** [comment]
### Exception when receiving the `@online` message:

```
2018-01-25 11:00:22.146 ERROR 1 --- [TaskExecutor-17] c.h.c.c.m.l.AbstractLifecycleService     : Exception during @online message handling:

com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "_internal" (class ch.hevs.cloudio.cloud.model.CloudioObject), not marked as ignorable (3 known properties: "objects", "conforms", "attributes"])
 at [Source: [B@4e799cd0; line: 1, column: 992] (through reference chain: ch.hevs.cloudio.cloud.model.Endpoint["nodes"]->java.util.LinkedHashMap["Xtender"]->ch.hevs.cloudio.cloud.model.Node["objects"]->java.util.LinkedHashMap["control"]->ch.hevs.cloudio.cloud.model.CloudioObject["_internal"])
```

**Comment by boozo-unlimited** [comment]
Fixed in 2ac3dab
